### PR TITLE
registration closed

### PR DESCRIPTION
**Pull request by tbushman**
If a githubUser is not already registered then redirect to '/'. The home page will have the message, 'Registration is closed.'

### CODE REVIEW

**Comment by raisedadead** [comment]
Works. Just a bit janky

image

**Comment by tbushman** [comment]
Too small ?

**Comment by raisedadead** [comment]
Don't worry I'll look into this. I am thinking of keeping it a bit flexible.

**Comment by raisedadead** [comment]
I'll update this PR.

**Comment by tbushman** [comment]
@raisedadead I added a session variable, regOpen. I tested it by deleting my user, clearing site data, directing to '/', then clicking sign in. This created a new user. I have `OPEN_FOR_REGISTRATION=false` in .env . What am I doing wrong?

**Comment by tbushman** [comment]
@raisedadead NVM it didn't create a new user so the session variable must not be working. One moment.

File: index.route.js [hunk 1]
```diff
@@ -21,6 +21,13 @@ api.use('/teams', teamRoutes);
 api.use('/webhook', webhookRoutes);
 
 router.get('/', (req, res) => {
+  if (process.env.OPEN_FOR_REGISTRATION) {
```
Comment (raisedadead) [inline index.route.js]:
```suggestion
  if (config.isOpenForRegistrations) {
```

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: index.route.js (hunk 1 above)
Comment (raisedadead) [reply index.route.js]: @tbushman I think you need to pick it up from the config object. That will fix it.

**Pull request merged**